lisa/tools/dmesg.py

```diff
@@ -33,8 +76,38 @@ def command(self) -> str:
     def _check_exists(self) -> bool:
         return True
 
-    def get_output(self, force_run: bool = False) -> str:
-        command_output = self._run(force_run=force_run)
+    def get_output(
+        self,
+        force_run: bool = False,
+        log_facility: Optional[List[str]] = None,
```

Comment:
Use an enum like other places, so it's easier to use. One level is enough, it should include the higher levels.

Reply:
So much nicer, when I was testing it wasn't consistent whether selecting a single level automatically gave you the levels above (looking at Ubuntu 22.04) but I can fix it up to do this

Reply:
Still testing though